operator/internal/central/extensions/reconcile_pvc.go

```diff
@@ -122,11 +190,20 @@ type reconcilePVCExtensionRun struct {
 	client           ctrlClient.Client
 	centralObj       *platform.Central
 	persistence      *platform.Persistence
+	defaultClaimSize resource.Quantity
 	defaultClaimName string
 	target           PVCTarget
 	log              logr.Logger
 }
 
+type Option func(*reconcilePVCExtensionRun)
```

Comment:
This package holds all extensions so this should be a `PVCOption` or `ReconcilePVCOption` to prevent ambiguity.